html/cross-origin-opener-policy/coep-navigate-popup.https.html

```diff
@@ -53,9 +53,9 @@
 
     async_test(t => {
       // For each test we open a COOP: same-origin/COEP: require-corp document in a popup and then
-      // navigate that to either a same-origin or same-site document whose COOP and COEP are set as
-      // per the top-most array. We then verify that this document has the correct opener for its
-      // specific setup.
+      // navigate that to either a document with same origin (site=="same-origin") or
```

Comment:
I think the original sentence might be clearer, but adding the parentheticals might be even better. This sentence however reads a little strange.

Reply:
Fixed.